cypress/page_objects/sales_invoices.js

```diff
@@ -1,6 +1,6 @@
 import Metasfresh from './page';
 
-class SalesInvoices extends Metasfresh {
+class _SalesInvoices extends Metasfresh {
```

Comment:
what's with the naming convention with underscores ?

Reply:
I chose underscore because it's the _not exported_ class (the internal one) the exported class, just below, has the normal name: ` SalesInvoices`. 

There are other classes, for example `page_objects\purchase_orders.js\purchaseOrders` which have imo a **really bad name** because they start with lowercase even though they're a class, and not just a "test variable".

Having the underscore in front of the internal class (eg. `_PurchaseOrders`) allows me to export that class in capital ("as it should be" imo): `export const PurchaseOrders = new _PurchaseOrders();`, and it also makes it way clearer that  "this is internal stuff, don't touch this".
I could chose to name the not-exposed class like `PurchaseOrderInternal` but i don't like that name, and think `_` symbolises better the internal part.

Reply:
Fair enough.